string GEOM_BOX="box"
string GEOM_SPHERE="sphere"
string GEOM_CYLINDER="cylinder"
string GEOM_CAPSULE="capsule"

string geom_type               # Type of object
geometry_msgs/Vector3 extents  # Extents of the box
float64 height                 # Height of cylinder and capsule
float64 radius                 # Radius of sphere, cylinder and capsule
geometry_msgs/Pose pose        # Pose to instantiate the object at
float64 mass                   # Mass of the object. 0 for fixed object
std_msgs/ColorRGBA color       # Color of the object. A=0 -> random color
string name                    # Id to assign to the object. Leave blank for automatic Id
---
bool success                   # Was the call successful
string error_msg               # Error message, if any
string object_id               # Object Id assigned to the loaded URDF